README.md

```diff
@@ -1,21 +1,24 @@
-# Setup of eWaterCycle system on SURF Research cloud
+# Instructions for system administrators to deploy the eWaterCycle platform
 
 ![Ansible Lint](https://github.com/eWaterCycle/infra/workflows/Ansible%20Lint/badge.svg)
 [![Concept DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1462548.svg)](https://doi.org/10.5281/zenodo.1462548)
 
-On the [SURF Research Cloud](https://servicedesk.surfsara.nl/wiki/display/WIKI/Research+Cloud+Documentation) an eWatercycle application will be available that when started will give
+This repo contains (codified) instructions for deploying the eWaterCycle platform. The target audience of these instructions are system administrators. For more information on the eWaterCycle platform (and how to deloy it) see the [eWaterCycle documentation[(https://ewatercycle.readthedocs.io/).
+
+For instructions on how to use the machine as deployed by this repo see the [User guide](USER.md).
+
+These instructions assume you have some basic knowledge of [vagrant](https://vagrantup.com) and
+ [Ansible](https://docs.ansible.com/ansible/latest/index.html).
+
+## Setup of eWaterCycle platform on the SURF Research cloud
+
+De environment used by the eWaterCycle platform development team is the [SURF Research Cloud](https://servicedesk.surfsara.nl/wiki/display/WIKI/Research+Cloud+Documentation). The setup instructions in this repo will create an eWatercycle application(a sort-of VM template) that when started will create a machine with:
 
 * Explorer: web visualization of available models / parameter sets combinations and a way to generate Jupyter notebooks
 * Jupyter Hub: to interactivly generate forcings and perform experiments on hydrological models using the [eWatercycle Python package](https://ewatercycle.readthedocs.io/)
 * ERA5 and ERA-Interim global climate data, which can be used to generate forcings
 * Installed models and their example parameter sets
 
```

Comment:
Maybe explain briefly that the one setting up the machine needs to have research budget, but can distribute access to the machine to others (using SRAM)?

Reply:
Added a note.